### ISSUE TITLE
Blaze compile errors completely silent if imported

### ISSUE THREAD

**Issue by TMeerhof** [meteor/meteor#10534]
In version 1.8 when compiling a blaze template from the `/imports` directory all errors are swallowed breaking the normal development process, because you get no line numbers or error specifications.

In 1.7 this problem is not present.

Reproduction:
Create a new meteor 1.8 project.

create a new malformed blaze template in the client folder.
```
<template name="test">
   {{#if}
</template>
```
You will get a compile error 
```
Errors prevented startup:
   
   While processing files with templating-compiler (for target web.browser):
   client/test.html:2: Expected `}}`
```

Now move the file to `/imports/client` and add the import statement `import '/imports/client/test.html';` to main.js. 

There is no compile time error, but when running the app you get the following run time error
`Uncaught Error: Cannot find module '/imports/client/test.html'`.

**Comment by TMeerhof** [comment]
this is also the case for .less files, here you will get the even more criptic error:
```
internal/process/next_tick.js:131
      callback();
```

**Comment by zenhack** [comment]
I'm still seeing this issue as of meteor 2.0; can we reopen this?

**Comment by haveaguess** [comment]
Agreed I have to refactor my files out of import to see what the error is and then refactor them back, very tedious

**Comment by StorytellerCZ** [comment]
Seems to be the same issue as in #10366

**Comment by StorytellerCZ** [comment]
@zodern do you have any plans for this issue?

**Comment by StorytellerCZ** [comment]
@zodern @jankapunkt maybe we should move this to the Blaze repository?

**Comment by jankapunkt** [comment]
I agree 👍
